fighthealthinsurance/admin.py

```
@@ -735,3 +737,235 @@ def has_change_permission(self, request, obj=None):
+        )
+
+    @admin.action(description="Purge selected fuzz attempts")
+    def purge_selected(self, request, queryset):
+        """Admin action to purge selected fuzz attempts."""
+        count = queryset.count()
+        # Delete associated files first
+        for obj in queryset:
+            if obj.encrypted_blob:
+                try:
+                    obj.encrypted_blob.delete(save=False)
+                except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)